kbase-extension/scss/partials/_narrative.scss

```diff
@@ -126,24 +126,31 @@
   table-layout: fixed;
 }
 
-.kb-staging-table-header__checkbox {
```

Comment:
I changed all of the header widths from % -> px and nested them under `.kb-staging-table .kb-staging-table-header` so that i could set padding for the first two